docs/copilot/ai-powered-suggestions.md

```diff
@@ -1,208 +1,93 @@
 ---
-Order: 
+Order: 3
 Area: copilot
-TOCTitle: AI-powered suggestions
+TOCTitle: AI code completions
 ContentId: 7ab2cd6c-45fd-4278-a6e8-1c9e060593ea
-PageTitle: AI-powered suggestions with GitHub Copilot
+PageTitle: AI-powered code completions with GitHub Copilot
 DateApproved: 02/28/2024
 MetaDescription: Enhance your coding with AI-powered suggestions from GitHub Copilot in Visual Studio Code.
 ---
-# GitHub Copilot in VS Code
+# Code completions with GitHub Copilot in VS Code
 
-The [GitHub Copilot](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) extension is an AI pair programmer tool that helps you write code faster and smarter. You can use the Copilot extension in Visual Studio Code to generate code, learn from the code it generates, and even configure your editor.
-
-With GitHub Copilot in VS Code you can:
-
-* Get inline code suggestions while you're writing and iterating on code.
-* Start a chat conversation to generate or refactor source code, produce documentation comments, or generate unit tests.
-* Ask questions to help ramp-up on a new code base, or accelerate learning a new programming language or framework.
-* Use chat features to discover and configure your VS Code setup.
-
-<iframe width="560" height="315" src="https://www.youtube.com/embed/Fi3AJZZregI" title="Get Started with the Future of Coding: GitHub Copilot" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+GitHub Copilot provides autocomplete-style code completions from an AI pair programmer as you code. As you start writing code or code-related items (comments, test, and more), Copilot presents suggestions automatically in the editor to help you code more efficiently. GitHub Copilot provides suggestions for numerous languages and a wide variety of frameworks.
 
 ## Getting started
 
-If you want to use GitHub Copilot, you either need a subscription for GitHub Copilot in your personal account, or you need to be assigned a seat by your organization.
-
-The following tutorials walk you through using GitHub Copilot in VS Code.
+> **Note**: After reviewing this topic, you can get started with the introductory [Copilot getting started tutorial](/docs/copilot/getting-started-chat.md) to get set up and take a hands-on tour of Copilot in VS Code.
```

Comment:
nit: should this state it's a hands-on tour of Copilot Chat?

```suggestion
> **Note**: After reviewing this topic, you can get started with the introductory [Copilot getting started tutorial](/docs/copilot/getting-started-chat.md) to get set up and take a hands-on tour of Copilot Chat in VS Code.
```

Reply:
The link was incorrect. It should actually point to the Getting started tutorial, not the Chat one. People on this article should have as a next step to get started with the intro tour.